.rubocop_todo.yml

```diff
@@ -411,6 +411,7 @@ RSpec/Capybara/FeatureMethods:
     - "spec/integration/receive_spec.rb"
     - "spec/integration/session_spec.rb"
     - "spec/integration/supplier_spec.rb"
+    - "spec/integration/home_spec.rb"
```

Comment:
could you add it in order? (then there is no diff when the file gets regenerated)